libs/librepcb/projecteditor/boardeditor/boardeditor.cpp

```diff
@@ -501,6 +503,12 @@ void BoardEditor::on_actionRebuildPlanes_triggered()
     }
 }
 
+void BoardEditor::on_actionAbout_triggered() noexcept
```

Comment:
This method is called two times when clicking on the menu item, so the dialog appears two times:
- One time from the `connect()` in the constructor
- One time from the implicit connection created by Qt because the method name has the pattern "on_WIDGETNAME_SIGNALNAME()".

I don't like the implicit connection because it is error prone. So better rename the method.

Another option would be to completely remove this method and provide a slot `about()` in the class `Application`, just like [`QApplication::aboutQt`](https://doc.qt.io/qt-5/qapplication.html#aboutQt). The pointer to the parent widget can be obtained with [`QApplication::activeWindow()`](http://doc.qt.io/qt-5/qapplication.html#activeWindow). This would simplify opening the about dialog to this:

    connect(mUi->actionAbout, &QAction::triggered, qApp, &Application::about);